src/main/java/org/quantumbadger/redreader/views/list/ListOverlayView.java

```diff
@@ -26,9 +27,24 @@
 import org.quantumbadger.redreader.common.AndroidApi;
 import org.quantumbadger.redreader.common.HandlerTimer;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
 // TODO add short highlight timer, add "onHighlight" and "onHighlightEnd" callbacks
 public class ListOverlayView extends FrameLayout {
 
+	public static final int TOUCH_EVENT_CLICK = 0;
+	public static final int TOUCH_EVENT_VERTICAL = 1;
+	public static final int TOUCH_EVENT_HORIZONTAL = 2;
+	public static final int TOUCH_EVENT_LONG_CLICK = 3;
+	public static final int TOUCH_EVENT_UNKNOWN = 4;
+	public static final int TOUCH_EVENT_SELECTED = 5;
```

Comment:
I think it would actually be `UNSELECTED` come to think of it. I found it rather strange that the enum was being compared to `null`. If you could have a look that would be awesome.